.gitlab-ci.yml

```diff
@@ -1,13 +1,35 @@
 stages:
+  - package
```

Comment:
I'm not 100 % sure how this works with the included files, but the included `packaging.yml` also has a `release` stage which I think should be added here.